docs/SETUP_REMOTE.md

```diff
@@ -29,6 +29,14 @@ There are other options like a Docker container per person though.
 
 Install tools you need to perform remote processing.
 
+If some tool require shell initialization scripts or special environment variables you should add them to your `~/.bashrc` or `~/.bash_profile` file depending on your remote operating system.
+
+Be sure to add all your configurations at the beginning of the file before this line :
+
+  ```bash
+  # If not running interactively, don't do anything
```

Comment:
Many distros do that, Amazon Linux for instance (based on Cent OS), most of Debian-like distros. Though not all of them have such a line.

Anyway I'd prefer to add this note since Ubuntu Server is very common choice.